README.md

```diff
@@ -1 +1,34 @@
-# mobile-menu-training
\ No newline at end of file
+
+
+# Mobile-Menu Training ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## Built With
+- HTML, CSS, JS
+- Linters, Node.js, Git
+
+To get a local copy up and running follow these simple example steps.
+- On your terminal and run the following command to clone the repository
+  
+  `git clone git@github.com:mateo951/mobile-menu-training.git`
+
```

Comment:
- [ ] Kindly fix this clone link in your Readme file, as you can see, this kind of link it's for users that have the correct public key for this repository. When you try to clone the repo without the key it throws the following error `git@github.com: Permission denied (publickey)`. It's better if you clone the repository directly from `https://github.com/mateo951/mobile-menu-training/`

Reply:
Done ✔ Thanks for the heads-up!

Reply:
Well addressed 👌👌